src/L10NSharp/UI/TipDialog.cs

```diff
@@ -25,10 +25,20 @@ public new Image Icon
 			set { _icon.Image = value; }
 		}
 
-		public static void Show(string message)
+		public static void ShowAltShiftClickTip(IWin32Window owner = null)
```

Comment:
Is this method expected to be called from clients, or should this be `internal` ?